packages/web/components/cards/generic-main-card.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactNode } from "react";
+
+import { IconLink } from "~/components/cards/icon-link";
+
+export const GenericMainCard: React.FC<{
+  children: ReactNode;
+  title?: string;
+  titleIcon?: ReactNode;
+  titleIconAction?: string;
+  width?: string;
+}> = ({ children, title, titleIcon, titleIconAction, width = "27" }) => {
+  return (
+    <div
+      className={`relative flex flex-col gap-8 overflow-hidden rounded-[24px] bg-osmoverse-800 px-1 py-1 lg:mx-auto md:mt-mobile-header md:gap-6 md:px-3 md:pt-4 md:pb-4`}
+      style={{ width: `${width}rem` }}
```

Comment:
@MaxMillington let's use Tailwind exclusively, instead of combining with inline styles.